client/app/bundles/course/survey/constants.js

```diff
@@ -96,7 +96,7 @@ const actionTypes = mirrorCreator([
 export const workflowStates = {
   Unstarted: 'unstarted',
   Attempting: 'attempting',
-  Submitted: 'submitted',
+  Published: 'published',
```

Comment:
can I check why was this changed to published?

Reply:
The submitted state is grey color for assessment as its not the final stage, but submitted is the final stage for surveys, so I changed it to have the green color:

Previous: ![image](https://user-images.githubusercontent.com/66063943/171796537-93d0e4d6-c04c-4d2e-a751-6ee3d578ea72.png)
Current: ![image](https://user-images.githubusercontent.com/66063943/171796429-0098005e-5210-41e3-9508-067a7ab169f4.png)

Or is there a better way of doing this?

Reply:
I think it is okay to leave the gray submitted colour consistent for both submissions as well as assessment.
No need to change.